### ISSUE TITLE
Listboxes don't read with NVDA in visual studio 2012

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3030]
**Reported by falinn.onda on 2013-02-28 21:29**
Hi,
I'm trying to debug with NVDA 2012.3.1 and visual studio 2012.
There are many important windows in debug view which I'm not able to read with NVDA using keyboard (see list below).

For example assume we do the following:run in debug view and when stopping in a breakpoints move the focus to "watch" window (CTRL+ALT+W, 1) after adding watch (context menu, add watch). In this case the focus is in the watch window but NVDA doesn't report the current line/item by NVDA+uparrow or NVDA+TAB or even with flat review/object navigation. moving with the arrows in the list doesn't read the watch list content.
If the mouse is placed on the item it does read the content.
However, when i press NVDA+numpadDivide to bring the mouse to focus it moves the mouse to incorrect location.
Using NVDA+uparrow says "blank" and NVDA+TAB says "watch 1" which is the name of the window.

This unfortunate situation repeats in the following windows:
locals, call stack, quick watch etc...

If I'm missing something and there is a way to work with VS2012 debugger (watches and similar), please let me know.
Help will be greatly appreciated.

**Comment by jsschultz** [comment]
@jcsteh @camlorn, the issue where the Autos, Locals, Watch, Quickwatch, and call stack windows are not being read by NVDA was on Microsoft's end and have been fixed in the latest release of Visual Studio 2017 RC. No more need to special case those windows :)

**Comment by feerrenrut** [comment]
Considering there is an open pull request, I think this issue should be looked at more closely. At least to make a plan on how to integrate and test the changes. While it sounds like this issue is fixed in Visual Studio 2017, there may be people still stuck working with earlier versions.

**Comment by leonardder** [comment]
Visual Studio has been superseded by 2013, 2015, 2017 and 2019. Given its age and the difficulty to maintain support for all these versions of Visual Studio, I'm closing this.
